src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -1,11 +1,19 @@
 import { Redirect, Route } from 'react-router-dom';
 import { useUser } from '../../context/UserContext';
 
-// A wrapper for <Route> that redirects to the login
-// screen if you're not yet authenticated.
 export default function PrivateRoute({ children, ...rest }) {
-  // TODO: Use the user in context to determine whether to redirect to /login
-  // const { user } = useUser();
+  const { user } = useUser();
 
-  return <Route {...rest} render={({ location }) => children} />;
+  return (
```

Comment:
Great logic here creating a user as a variable to see where to send the user inside your private route